src/app/modules/submission/email-submission/email-submission.controller.ts

```diff
@@ -317,6 +321,30 @@ const submitEmailModeForm: ControllerHandler<
             meta: logMetaWithSubmission,
           })
 
+          // TODO 6395 make responseMetadata mandatory
+          if (responseMetadata) {
+            // response time
+            statsdClient.distribution(
+              'formsg.submissions.responseTimeMetadata',
```

Comment:
I think you can drop the suffix `Metadata`, the metric is just tracking `responseTime` :)
```suggestion
              'formsg.submissions.responseTime',
```

And same comment for al the other metrics in the PR of course :) 

Also do note: metrics names and metrics tag names get lowercased in DD, so best to have lowercased in code as well, so we can see whether they read well, and we can add additional separators (dots or underscores) when they do not.

Reply:
[87c6687](https://github.com/opengovsg/FormSG/pull/6406/commits/87c6687fb1421405a8da2352642ac697f9984f24)

Thanks Tim!

Reply:
I also abstracted out the prefix here: [5c71d4a](https://github.com/opengovsg/FormSG/pull/6406/commits/5c71d4a49157d3deae2d27a9d179c2f5e93de222)